app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file, file_to_copy = command.split()
+    if command_name != "cp":
+        print("This command not exist")
+        return
+    try:
+        with (open(file, "r") as current_file,
+              open(file_to_copy, "x") as new_file):
+            new_file.write(current_file.read())
+    except FileNotFoundError:
+        print(f"There is not file with name {file}")
+        return
+    except FileExistsError:
+        print("This file already exist")
```

Comment:
Are you sure your custom messages in console are more descriptive than default ones?

Reply:
I thought so